src/components/context/AuthContext.ts

```diff
@@ -6,7 +6,7 @@ import createDataContext from './createDataContext';
 const authReducer = (state, action) => {
   switch (action.type) {
     case 'signin':
-      return { ...state, isSignedIn: true, token: action.payload.token, user: action.payload.user };
+      return { ...state, isSignedIn: true, token: action.payload.token, user: action.payload.user, staySignedIn: action.payload.staySignedIn };
```

Comment:
Great energy! But AuthContext dosen't require this data, please revert authContext related changes as this is a front-end only issue. You'll have an opportunity to do the corresponding back-end changes on the rails side [here](https://github.com/rubyforgood/casa/issues/6210)